modules/dxkultureBidAdapter.js

```diff
@@ -138,33 +138,92 @@ export const spec = {
 
   getUserSyncs: function (syncOptions, serverResponses, gdprConsent, uspConsent) {
     logInfo('dxkulture.getUserSyncs', 'syncOptions', syncOptions, 'serverResponses', serverResponses);
-
     let syncs = [];
 
     if (!syncOptions.iframeEnabled && !syncOptions.pixelEnabled) {
       return syncs;
     }
 
-    if (syncOptions.iframeEnabled || syncOptions.pixelEnabled) {
-      let pixelType = syncOptions.iframeEnabled ? 'iframe' : 'image';
-      let queryParamStrings = [];
-      let syncUrl = SYNC_URL;
-      if (gdprConsent) {
-        queryParamStrings.push('gdpr=' + (gdprConsent.gdprApplies ? 1 : 0));
-        queryParamStrings.push('gdpr_consent=' + encodeURIComponent(gdprConsent.consentString || ''));
-      }
-      if (uspConsent) {
-        queryParamStrings.push('us_privacy=' + encodeURIComponent(uspConsent));
+    serverResponses.forEach(resp => {
+      const userSync = deepAccess(resp, 'body.ext.usersync');
+      if (userSync) {
+        let syncDetails = [];
+        Object.keys(userSync).forEach(key => {
+          const value = userSync[key];
+          if (value.syncs && value.syncs.length) {
+            syncDetails = syncDetails.concat(value.syncs);
+          }
+        });
+        syncDetails.forEach(syncDetails => {
+          let queryParamStrings = [];
+          let syncUrl = syncDetails.url;
+
+          if (syncDetails.type === 'iframe') {
+            if (gdprConsent) {
+              queryParamStrings.push('gdpr=' + (gdprConsent.gdprApplies ? 1 : 0));
+              queryParamStrings.push('gdpr_consent=' + encodeURIComponent(gdprConsent.consentString || ''));
+            }
+            if (uspConsent) {
+              queryParamStrings.push('us_privacy=' + encodeURIComponent(uspConsent));
+            }
+            syncUrl = `${syncDetails.url}${queryParamStrings.length > 0 ? '?' + queryParamStrings.join('&') : ''}`
+          }
+
+          syncs.push({
+            type: syncDetails.type === 'iframe' ? 'iframe' : 'image',
+            url: syncUrl
+          });
+        });
+
+        if (syncOptions.iframeEnabled && !syncOptions.pixelEnabled) {
+          syncs = syncs.filter(s => s.type == 'iframe');
+        }
+
+        if (!syncOptions.iframeEnabled && syncOptions.pixelEnabled) {
+          syncs = syncs.filter(s => s.type == 'image');
+        }
```

Comment:
I also dont think this is right which is why I suggested using an if/else for whichever should take priority. Not this suggests that if both pixel and iframe are enabled, neither condition will be true. Can you please update to an if/else giving one type priority over the other i.e. `if (syncOptions.iframeEnabled) {...} else if (syncOptions.pixelEnabled) {...}` which would give priority to iframe syncing or vice versa.